HarmonyX2Interop/HarmonyX2Interop.csproj

```diff
@@ -1,118 +1,23 @@
-          <HintPath>$(SolutionDir)\packages\Mono.Cecil.0.10.4\lib\net35\Mono.Cecil.Rocks.dll</HintPath>
-          <Private>True</Private>
-        </Reference>
-        <Reference Include="MonoMod.RuntimeDetour, Version=21.8.5.1, Culture=neutral, PublicKeyToken=null">
-          <HintPath>$(SolutionDir)\packages\MonoMod.RuntimeDetour.21.8.5.1\lib\net35\MonoMod.RuntimeDetour.dll</HintPath>
-          <Private>True</Private>
-        </Reference>
-        <Reference Include="MonoMod.Utils, Version=21.8.5.1, Culture=neutral, PublicKeyToken=null">
-          <HintPath>$(SolutionDir)\packages\MonoMod.Utils.21.8.5.1\lib\net35\MonoMod.Utils.dll</HintPath>
-          <Private>True</Private>
-        </Reference>
-        <Reference Include="System" />
-        <Reference Include="System.Core" />
-        <Reference Include="System.Data" />
-        <Reference Include="System.Xml" />
-    </ItemGroup>
-    <ItemGroup>
-        <Compile Include="Attributes.cs" />
-        <Compile Include="Exceptions.cs" />
-        <Compile Include="GlobalSuppressions.cs" />
-        <Compile Include="Harmony.cs" />
-        <Compile Include="HarmonyMethod.cs" />
-        <Compile Include="Internal\AccessCache.cs" />
-        <Compile Include="Internal\EmitterExtensions.cs" />
-        <Compile Include="Internal\GlobalPatchState.cs" />
-        <Compile Include="Internal\ILManipulator.cs" />
-        <Compile Include="Internal\Legacy.cs" />
-        <Compile Include="Patching\Patch.cs" />
-        <Compile Include="Patching\Patches.cs" />
-        <Compile Include="Patching\PatchProcessor.cs" />
-        <Compile Include="Patching\ReversePatcher.cs" />
-        <Compile Include="Properties\AssemblyInfo.cs" />
-        <Compile Include="Tools\CodeMatcher.cs" />
-        <Compile Include="Tools\Extensions\CodeInstructionExtensions.cs" />
-        <Compile Include="Tools\Extensions\CollectionExtensions.cs" />
-        <Compile Include="Tools\Extensions\GeneralExtensions.cs" />
-        <Compile Include="Tools\Extensions\SymbolExtensions.cs" />
-        <Compile Include="Tools\FileLog.cs" />
-        <Compile Include="Tools\JetBrains.Annotations.cs" />
-        <Compile Include="Tools\Logger.cs" />
-        <Compile Include="Tools\Reflection\AccessTools.cs" />
-        <Compile Include="Tools\Reflection\DelegateTypeFactory.cs" />
-        <Compile Include="Tools\Reflection\FastAccess.cs" />
-        <Compile Include="Tools\Reflection\MethodInvoker.cs" />
-        <Compile Include="Tools\Reflection\Traverse.cs" />
-        <Compile Include="Transpiling\CodeInstruction.cs" />
-        <Compile Include="Transpiling\ExceptionBlock.cs" />
-        <Compile Include="Transpiling\Transpilers.cs" />
-    </ItemGroup>
-    <ItemGroup>
-      <None Include="packages.config" />
+        <ProjectReference Include="..\HarmonyXInterop\HarmonyXInterop.csproj" ExcludeAssets="compile"/>
```

Comment:
Actually, I'm not sure about that one. ProjectReference does not officially support ExcludeAssets